specification.markdown

```diff
@@ -778,3 +774,47 @@ minOccurs: 0
 type: Binary
 
 description: Used to void damaged data, to avoid unexpected behaviors when using damaged data. The content is discarded. Also used to reserve space in a sub-element for later use.
+
+# IANA Considerations
+
+This document creates a new IANA Registry called
+"CELLAR EBML Element ID Registry".
+
+Element IDs are described in section {{#element-id}}.  Element IDs are
+encoded using the VINT mechanism described in
+section {{#variable-sized-integer}} can be between one and five bytes
+long. Five byte long Element IDs are possible only if declared in the header.
+
+One byte Element IDs are numbers between 1 and 126. These items are valuable
+because they short, and need to be used for commonly repeated elements.
+Values from 1 to 126 are to be allocated according to RFC Required.
+
+Two byte Element IDs are numbers between 127 and 16382.
+Numbers may be allocated within this range according to Specification Required.
+
+Three byte Element IDs are numbers between 16383 and 2097150.
+Numbers may be allocated within this range according to First Come First Served.
+
+Four byte Element IDs are numbers between 2097151 and 268435456.
+Four byte Element IDs are somewhat special in that they are useful for
+resynchronizing to major structures in the event of data corruption or
+loss.  As such four byte Element IDs are split into two categories.
+Four byte Element IDs whose lower three bytes (as encoded) would make
+printable 7-bit ASCII values may be allocated only Specification Required.
```

Comment:
Should define the ascii range specifically here. For instance the lower three bytes of the EBML Element are 0x45DFA3 which is extended ascii for Eß£ (~'ebml')

Reply:
ok i see you define a range below